src/serac/physics/thermal_conduction_functional.hpp

```diff
@@ -127,8 +127,8 @@ class ThermalConductionFunctional : public BasePhysics {
                                                                .vector_dim = 1,
                                                                .ordering   = mfem::Ordering::byNODES,
                                                                .name       = detail::addPrefix(name, "temperature")})),
-        M_functional_(&temperature_.space(), &temperature_.space()),
-        K_functional_(&temperature_.space(), &temperature_.space()),
+        M_functional_(&temperature_.space(), {&temperature_.space()}),
```

Comment:
Having to have the brackets is awkward here.  Can we add a constructor to avoid it?

Reply:
Are you requesting an extra constructor when there is only 1 trial-space case that looks like

```cpp
serac::Functional(test_space, trial_space)
```

or asking that the existing constructor use a variadic list, rather than `std::array` for its trial spaces? e.g.

```cpp
serac::Functional(test_space, trial_space_0, trial_space_1, ... trial_space_n)
```

(instead of the current):

```cpp
serac::Functional(test_space, {trial_space_0, trial_space_1, ... trial_space_n})
```

Reply:
Okay I understand now.  I missed the first parameter was a `test_space` with an array of `trial_space`s.